java/org/apache/catalina/startup/Tomcat.java

```diff
@@ -1303,7 +1303,7 @@ public static void main(String[] args) throws Exception {
                 break;
             }
         }
-        org.apache.catalina.startup.Tomcat tomcat = new org.apache.catalina.startup.Tomcat();
+        Tomcat tomcat = new Tomcat();
```

Comment:
The removal of the FQCN is fine but as the only remaining change is rather small for the effort required to review and merge a PR. A PR that addressed this for the entire code base (I believe there is a PMD rule to check for this) would be useful.